### ISSUE TITLE
Leading zero dropped when using meta-data editor.

### ISSUE THREAD

**Issue by CptMcSplody** [loot/loot#1056]
So if I add a `CRC` with a leading zero through LOOT to cleaning info it will save correctly and show up in the `userlist.yaml` with the full `CRC`.

![image](https://user-images.githubusercontent.com/1944639/49289743-ac719300-f49c-11e8-91e7-bc4884b82355.png)

But if I reopen the editor, the leading zero is removed from the `CRC` in the `userlist.yaml`.

![image](https://user-images.githubusercontent.com/1944639/49289804-e0e54f00-f49c-11e8-854d-caa9f4dbe4e0.png)

**Comment by Ortham** [comment]
Yes, it's the same as in #290. It's a harmless consequence of storing the value as a number and not padding it out when it gets put in the input, but that should probably happen for consistency.

**Comment by CptMcSplody** [comment]
Okay, will I mark this as a duplicate and close?

**Comment by Ortham** [comment]
No, it's not a duplicate, the other issue is fixed, this is the same thing
happening somewhere else.
